### PR TITLE
VIH-6162 Added Video Api Nuget Client

### PR DESCRIPTION
**Pull request by npoxon**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-6162

### Change description ###
- Fixes build issues after video api changes

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by williambowshersolirius — DISMISSED** [review]

**Review by shaed-parkar — DISMISSED** [review]

File: AdminWebsite/AdminWebsite.UnitTests/Controllers/HealthTests.cs [hunk 1]
```diff
@@ -156,7 +156,7 @@ public async Task Should_return_internal_server_error_result_when_notification_a
         }
 
         [Test]
-        public async Task Should_return_notifiy_exception_when_notification_api_not_found()
+        public async Task Should_return_notify_exception_when_notification_api_not_found()
```
Comment (levibutler) [inline AdminWebsite/AdminWebsite.UnitTests/Controllers/HealthTests.cs:159]: good spot!

File: AdminWebsite/AdminWebsite.AcceptanceTests/Hooks/DriverHooks.cs [hunk 2]
```diff
@@ -67,12 +67,11 @@ public void SetTimeZone(TestContext context)
             context.TimeZone = new TimeZone(context.WebConfig.SauceLabsConfiguration.RunningOnSauceLabs(), context.WebConfig.TestConfig.TargetOS);
         }
 
-        private static bool EnableLogging(ScenarioInfo scenario)
+        private static bool EnableLogging()
         {
-            return !scenario.Tags.Contains("DisableLogging");
+            return false;
```
Comment (npoxon) [inline AdminWebsite/AdminWebsite.AcceptanceTests/Hooks/DriverHooks.cs:72]: Also I am disabling logging as part of the AC tests to see if it fixes a strange Sauce Labs issue. @robin-weller is aware

**Review by levibutler — APPROVED** [review]

File: AdminWebsite/AdminWebsite.UnitTests/Controllers/HealthTests.cs (hunk 1 above)
Comment (npoxon) [reply AdminWebsite/AdminWebsite.UnitTests/Controllers/HealthTests.cs:159]: :)

**Pull request merged**